cmd/stellar-rpc/internal/xdr2json/conversion.go

```diff
@@ -63,16 +63,16 @@ func convertAnyBytes(xdrTypeName string, field []byte) (json.RawMessage, error)
 	// scope just added to show matching alloc/frees
 	{
```

Comment:
I think the scope is not necessary anymore now that you're using defer to free the memory